src/core/services/download.js

```diff
@@ -0,0 +1,12 @@
+export default function download(data, textMimeType, fileName) {
+	const csvFile = new Blob([data], {type: `text/${textMimeType}`});
```

Comment:
remove default and add export of download.js to index.js
also want to have next contract:
```javascript
download(name, data, mimeType)...
```